### PR TITLE
#364: WHED appendix title handling rewritten to complement #359

### PR DESCRIPTION
**Pull request by gregoryfoster**
Based on @cmc333333's [comments](https://github.com/eregs/regulations-parser/pull/365#issuecomment-290421748) on #365, I've added a `get_appendix_title` function to `regparser/tree/gpo_cfr/appendices.py`, removed `get_app_title` from `interpparser/gpo_cfr.py`, and replaced all references.

### CODE REVIEW

File: interpparser/gpo_cfr.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from regparser.tree.depth import markers as mtypes
 from regparser.tree.depth import heuristics, rules
 from regparser.tree.depth.derive import derive_depths
-from regparser.tree.gpo_cfr.appendices import appendix_headers
+from regparser.tree.gpo_cfr import appendices
```
Comment (gregoryfoster) [inline interpparser/gpo_cfr.py]: I attempted to only import the required function (`get_appendix_title`) but my environment complained so I imported the entire module.

File: interpparser/gpo_cfr.py [hunk 2]
```diff
@@ -292,26 +292,14 @@ def per_node(node):
...
-
-def get_app_title(node):
-    """ Appendix/Supplement sections have the title in an HD tag, or
-    if they are reserved, in a <RESERVED> tag. Extract the title. """
-
-    titles = node.xpath("./HD[@SOURCE='HED']")
-    if titles:
-        return titles[0].text
-    else:
-        return node.xpath("./RESERVED")[0]
```
Comment (gregoryfoster) [inline interpparser/gpo_cfr.py]: Replaced by [`get_appendix_title`](https://github.com/eregs/regulations-parser/pull/367/commits/a389c4b75dafeb45aeb3ae337c2aa042e42e5b61#diff-ea3142a6202d703dc2e030594367c8dcR52).

**Comment by cmc333333** [comment]
Code looks good to me, though the linter's complaining that you need a newline above `get_appendix_title`'s definition. Happy to do that for you (there should be a checkbox on your PR to allow maintainers to edit your PR) if you don't have time.

**Comment by gregoryfoster** [comment]
Thank you!  Can you point me to how the project is linting code?

**Comment by cmc333333** [comment]
Excellent, thanks! We're currently [installing a bunch of flake8 plugins](https://github.com/eregs/regulations-parser/blob/0c650cda3e09c8abd9485133a65ec255357f7c8b/requirements_test.txt) and then [running `flake8 .` in travis](https://github.com/eregs/regulations-parser/blob/0c650cda3e09c8abd9485133a65ec255357f7c8b/test-travis.sh#L6).

**Comment by cmc333333** [comment]
Thanks @gregoryfoster !

**Pull request merged**